test/infrastructure/docker/internal/controllers/backends/inmemory/inmemorymachine_backend.go

```diff
@@ -256,8 +142,8 @@ func (r *InMemoryMachineReconciler) reconcileNormalCloudMachine(ctx context.Cont
 
 	// Wait for the VM to be provisioned; provisioned happens a configurable time after the cloud machine creation.
 	provisioningDuration := time.Duration(0)
-	if inMemoryMachine.Spec.Behaviour != nil && inMemoryMachine.Spec.Behaviour.VM != nil {
-		x := inMemoryMachine.Spec.Behaviour.VM.Provisioning
+	if inMemoryMachine.Spec.Backend.InMemory.VM != nil {
```

Comment:
Looks like we dropped the "Behavior" level. I assume this was intended?

Reply:
Yes this was intentional, but let me know if you prefer to bring it back